modules/win-instance/README.md

```diff
@@ -1,4 +1,49 @@
+
```

Comment:
What's up with the leading blank lines?

Reply:
the leading blank lines are where I defined the `modules` dictionary. Moving all maps/dictionaries to a separate JSON file.

Reply:
Oh, did you try `{%- ... -%}`? Note the `-` in particular. That should tell jinja to exclude/remove any leading and trailing whitespace.

Reply:
`{%- ... -%}` works. thanks!